www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -18,6 +18,14 @@
     </webgui>
 
     <http>
+      <workerprocesses type="IntegerField">
+        <default>1</default>
+        <Required>Y</Required>
+      </workerprocesses>
+      <workerconnections type="IntegerField">
+        <default>1024</default>
+        <Required>Y</Required>
+      </workerconnections>
```

Comment:
I would define a minimum (1) and this must include a version bump. => 1.17.0

Reply:
Done